KomaMRICore/src/datatypes/sequence/Grad.jl

```diff
@@ -184,7 +184,6 @@ end
 	N, M = size(GR)
 	[sum(A[i,1:N] .* GR[:,j]) for i=1:N, j=1:M]
 end
-zero(::Grad) = Grad(0,0)
```

Comment:
Pretty sure this is used for Gradient rotation (multiplication with a rotation matrix).

Reply:
I added the function again. Apparently it's necessary to add Base.zero in order to make it work.

Reply:
But was it necessary? my hunch is that it was, but maybe it wasn't